README.md

```diff
@@ -33,6 +44,17 @@ $GLOBALS['TYPO3_CONF_VARS']['LOG']['writerConfiguration'][\TYPO3\CMS\Core\Log\Lo
 ];
 ```
 
+If you want to have different environments to filter by in Sentry, you can set them like this:
+```php
+$GLOBALS['TYPO3_CONF_VARS']['SYS']['environment'] = 'development';
+```
+
+## Where should I put the configuration?
```

Comment:
I like that you added a paragraph on this topic. That I disagree with *what* you wrote shows me that it is worth mentioning in the Readme.

This package expects settings to be exposed in main TYPO3 configuration (aka `TYPO3_CONF_VARS`). How you expose it, totally depends on your use case, your deployment strategy and project structure.

It does **not** depend on whether you want to use "only during development" and by all means, you should **never** expose the configuration by an extension's `ext_localconf.php` file, as these files are evaluated pretty late in TYPO3's boot process and would mean, you'd miss out all errors that happen *before* this step.

This means: Put the configuration in either `LocalConfiguration.php` or `AdditionalConfiguration.php` depending on your needs and structure of your project.

Reply:
Done. Thx for your valuable feedback!